src/gui/LmmsStyle.cpp

```diff
@@ -45,11 +45,11 @@ QLinearGradient getGradient( const QColor & _col, const QRectF & _rect )
 	qreal saturation = _col.saturationF();
 
 	QColor c = _col;
-	c.setHsvF( hue, 0.42 * saturation, 0.98 * value ); // TODO: pattern: 1.08
+	c.setHsvF( hue, 0.42 * saturation, 0.98 * value ); // TODO: clip: 1.08
```

Comment:
I really wonder if they meant "LMMS pattern" or some kind of different pattern with it. I guess the former, because gradients can be relevant for patterns. Though, in this case, I'd use "MIDI clip"?